src/tests/var/test06.vtc

```diff
@@ -0,0 +1,38 @@
+varnishtest "Test that running out of workspace memory fails the request"
+
+server s1 {
+	rxreq
+	txresp
+} -start
+
+# This bit was tricky to get right:
+#
+# * we are limiting the workspace size to 9k (empirically determined minimum),
```

Comment:
you should be able to do something a bit simpler using `.workspace_alloc()` from `vmod_vtc` (in core `varnish`)

Reply:
Thanks for pointing this out, I completely missed the existence of vmod vtc!

Using `vtc.workspace_alloc()` is much cleaner than guessing at the workspace size